admin/src/react-components/service-editor.js

```diff
@@ -124,6 +124,14 @@ function isEmptyObject(obj) {
   return true;
 }
 
+function getConfigurables(categorySchema) {
+  return getDescriptors(categorySchema).filter(
+    ([path, descriptor]) =>
+      (qs.get("show_internal_configs") !== null || descriptor.internal !== "true") &&
+      (qs.get("show_oidc_configs") !== null || path[1] !== "oidc")
```

Comment:
Why was it necessary to add another query param for this? Isn't internal_configs sufficient?